torso-bundle.js

```diff
@@ -488,7 +488,7 @@
           }
           return $.ajax({
               type: collection.fetchHttpAction,
-              url: collection.url + collection.getByIdsUrl,
+              url: _.result(collection, 'url') + collection.getByIdsUrl,
```

Comment:
while i want this change very much, you have it just in the bundle. Mind adding it to the collection mixin file as well?